Core/ModuleInstaller.cs

```diff
@@ -890,28 +890,39 @@ private void Uninstall(string modName)
         /// Takes a collection of directories and adds all parent directories within the GameData structure.
         /// </summary>
         /// <param name="directories">The collection of directory path strings to examine </param>
-        private HashSet<string> AddParentDirectories(HashSet<string> directories )
+        public HashSet<string> AddParentDirectories(HashSet<string> directories)
         {
             var newDirectories = new HashSet<string>();
-            foreach (string directory in directories)
+            if (directories == null)
             {
-                if ( directory.Contains(ksp.GameData()))
+                return newDirectories;
+            }
+
+            var normalGameData = KSPPathUtils.NormalizePath(ksp.GameData());
```

Comment:
ksp.GameData() is already normalized 

        public string GameData()
        {
            return KSPPathUtils.NormalizePath(
                Path.Combine(GameDir(), "GameData")
            );
        }